Console_File_Maneger/ConsoleUserInerface.cs

```diff
@@ -0,0 +1,186 @@
+            }
+            else
+            {
+                Wind = DisplayConsole.WindowsWidth / 2 + 1;
+            }
+            for (int i = (DisplayConsole.Line1_1 - 2) * page; i < DataDirs[NumDisplay].AllDirectoris.Length; i++, poz++)
+            {
+                if (poz <= DisplayConsole.Line1_1 - 1)
+                {
+                    if (i <= DataDirs[NumDisplay].Dirs_Info.Length)
+                    {
+                        DisplayConsole.ChangrForegroundColor(Color.White);
+                    }
+                    else
+                    {
+                        DisplayConsole.ChangrForegroundColor(Color.Yellow);
+                    }
+                    DisplayConsole.PrintWrite(Wind, poz, DataDirs[NumDisplay].AllDirectoris[i]);
+                }
+                else
+                {
+                    break;
+                }
+            }
+            DisplayConsole.Print_Cursor(DataDirs[DataDirectores.Select_Window].AllDirectoris, 0, DataDirs);
+        }
+        public void PrintChangeWind()
+        { }
+        //=========== Clear ==================
+        public void ClearAll()
+        {
+            ClearLeftWindow();
+            ClearRightWindow();
+            ClearInfoWindow();
+            ClearCommandWindow();
+            ClearPages();
+            ClearError();
+        }
+        public void ClearLeftWindow()
+        {
+            ClearWindows(0);
+        }
+        public void ClearRightWindow()
+        {
+            ClearWindows(1);
+        }
+        public void ClearInfoWindow()
+        {
+            DisplayConsole.CursorVisible(false);
+            for (int i = 1; i < DisplayConsole.WindowsWidth - 2; i++)
+            {
+                for (int j = DisplayConsole.Line1_2 + 1; j < DisplayConsole.Line2_1; j++)
+                {
+                    DisplayConsole.SetCursorPosition(i, j);
+                    DisplayConsole.PrintWrite(" ");
+                }
+            }
+        }
+        public void ClearCommandWindow()
+        {
+            DisplayConsole.DelitConsole();
+        }
+        public void ClearPages()
+        {
+        }
+        public void ClearError()
+        {
+            ClearInfoWindow();
+        }
+        public void ClearWindows(int NumDisp)
+        {
+            DisplayConsole.CursorVisible(false);
+            DisplayConsole.ChangeBackgroundColor(Color.Black);
+            int start, end;
+            if (NumDisp == 0)
+            {
+                start = 1;
+                end = (DisplayConsole.WindowsWidth - 1) / 2;
+            }
+            else
+            {
+                end = DisplayConsole.WindowsWidth - 1;
+                start = DisplayConsole.WindowsWidth / 2;
+            }
+            for (int i = start; i < end; i++)
+            {
+                for (int j = 1; j < DisplayConsole.Line1_1; j++)
+                {
+                    DisplayConsole.PrintWrite(i, j, " ");
+                }
+            }
```

Comment:
```suggestion
            for (int i = start; i < end; i++)
                for (int j = 1; j < DisplayConsole.Line1_1; j++)
                    DisplayConsole.PrintWrite(i, j, " ");
```
Куча фигурных скобок, которые не несут никакой логической нагрузки - можно сократить сделав код более убористым.